packages/components/src/components/buttons/PinButton/PinButton.tsx

```diff
@@ -1,30 +1,37 @@
+import {
+    Elevation,
+    mapElevationToBackground,
+    mapElevationToBorder,
+    spacingsPx,
+} from '@trezor/theme';
 import { ButtonHTMLAttributes } from 'react';
 import styled from 'styled-components';
+import { useElevation } from '../../ElevationContext/ElevationContext';
 
-const Button = styled.button`
+const Button = styled.button<{ $elevation: Elevation }>`
     max-width: 100px;
     height: 60px;
     transition: all 0.2s;
     position: relative;
     cursor: pointer;
-    margin: 4px;
+    margin: ${spacingsPx.xxs};
 
     width: 100%;
 
     border-radius: 5px;
```

Comment:
border-radius: ${borders.radii.xxs};

Reply:
That is 4px, not 5 as originally. So I wasn't changing it. Is that really what we want?

Reply:
Yes, it's fine